docs/zkapps/faq.mdx

```diff
@@ -111,6 +111,23 @@ this.num.assertEquals(currentState);
 
 This ensures that the transaction fails if the value of the field in question has changed.
 
+### How do I restrict a provable method so that it can be invoked only by a specific address?
+
+There are a number of ways to accomplish this:
+- Provably add a signed child account update. Then, the method can be invoked only with a signature from the predefined address.
+```TypeScript
+// in your method create empty signed account update
+AccountUpdate.createSigned(someAddress);
+```
+
+```TypeScript
+// later, sign the tx
+tx.sign([privateKeyOfAddress]);
+```
+
+- Require a private key as a private input, and check that `privateKey.toPublicKey().assertEquals(this.address)`
+- Require a signature as an input.
```

Comment:
by `Require a signature as an input.` do we mean to use the default value of signature? like https://docs.minaprotocol.com/zkapps/o1js/permissions#default-permissions 

or do we want to set explicitly set the input field permission to signature? (or something else entirely?)

I was looking for a doc to link to, and found out that I need to learn more about what this means.

Permissions doc https://docs.minaprotocol.com/zkapps/o1js/permissions

> `signature`: Fields that have their permission set to `signature` can only be manipulated by account updates that are accompanied and authorized by a valid signature.

Or do we mean something as described in https://docs.minaprotocol.com/zkapps/o1js/interact-with-mina#signing-transactions-and-explicit-account-updates ?

> Signature authorization – used to update user accounts. Signatures are verified against the account's public key.